flask_monitoringdashboard/core/reporting/questions/median_latency.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+from scipy.stats import median_test
+
+from flask_monitoringdashboard.core.reporting.questions.report_question import (
+    ReportAnswer,
+    ReportQuestion,
+)
+from flask_monitoringdashboard.database import session_scope
+
+from flask_monitoringdashboard.database.request import get_latencies_sample
+
+
+class MedianLatencyReportAnswer(ReportAnswer):
+    def __init__(
+        self,
+        is_significant,
+        comparison_interval_latencies_sample=None,
```

Comment:
should we maybe start adding type annotations?

Reply:
I would love that, but we still support Python 2.7, right? I don't think that is supported in Python 2.7.

In that case, is it an idea to drop Python 2.7 support soon?

Reply:
ah, you're right; forget about it for now :)